examples/fungible/README.md

```diff
@@ -13,7 +13,7 @@ instance represents a different fungible token.
 
 Individual chains have a set of accounts, where each account has an owner and a balance. The
 same owner can have accounts on multiple chains, with a different balance on each chain. This
-means that an account's balance is sharded across one or more chains.
+means that an account's balance is shared across one or more chains.
```

Comment:
I think here we probably actually meant "sharded". But maybe "distributed" would be a better word?